### PR TITLE
Sortable attributes in http-ui

### PR DESCRIPTION
**Pull request by irevoire**
- Add a `reset_sortable_attribute` method
- Add the `sortable_attributes` to http-ui
- Fix some broken test in http-ui

### CODE REVIEW

**Comment by irevoire** [comment]
The tests I fixed were broken since multiple PRs. 
I think we should either add them to the CI or delete them entirely (to me, they seem useless?)

File: http-ui/src/main.rs [hunk 1]
```diff
@@ -448,6 +451,15 @@ async fn main() -> anyhow::Result<()> {
                             Setting::NotSet => (),
                         }
 
+                        // We transpose the settings JSON struct into a real setting update.
+                        match settings.sortable_attributes {
+                            Setting::Set(sortable_attributes) => {
+                                builder.set_sortable_fields(sortable_attributes)
+                            }
+                            Setting::Reset => builder.reset_sortable_fields(),
```
Comment (Kerollmops) [inline http-ui/src/main.rs:459]: For the moment you could rebase your branch on https://github.com/meilisearch/milli/pull/330 to use this `reset` method.

**Review by Kerollmops — CHANGES_REQUESTED** [review]
Whoops, could you remove your first commit as it is already implemented in https://github.com/meilisearch/milli/pull/330.

**Comment by irevoire** [comment]
Ah yes, sorry I totally forgot about this PR (even though I reviewed it :see_no_evil: )
And what do you thinks of the tests?

**Comment by Kerollmops** [comment]
Those tests are good, thank you for fixing them! :)

File: http-ui/src/main.rs (hunk 1 above)
Comment (irevoire) [reply http-ui/src/main.rs:459]: done

**Review by Kerollmops — APPROVED** [review]
LGTM!
bors merge

**Pull request merged**